src/debug/daccess/nidump.cpp

```diff
@@ -2182,6 +2182,10 @@ void NativeImageDumper::FixupBlobToString(RVA rva, SString& buf)
         buf.Append( W("Statics ID for ") );
         goto EncodeType;
 
+    case ENCODE_GCPOLL_TRAP_STATIC:
```

Comment:
If you end up doing this, this should be encoded as `READYTORUN_HELPER_XXX`. There are several similar ones: `READYTORUN_HELPER_Module`, `READYTORUN_HELPER_GSCookie`.

Reply:
Oops. I am not going down that path. It is going to require reworking the entire GCPoll insertion logic.